### ISSUE TITLE
Management Zones

### ISSUE THREAD

**Issue by PatrikSteuer** [Dynatrace/ufo-esp32#25]
### Use Case
As an SE team I would like to monitor entities within an environment which we are responsible for.
### Meta Info
There are multiple Systems reporting into the same Dynatrace environment, seperated by management zones. 
It would be nice to have the possibility to configure 2 management zones, or even two environments, so that each ufo ring could monitor show the state of one configuration.

**Comment by qdengr10** [comment]
Hi @helmut-s 
Sorry to dig up this topic ; I am currently playing with the UFO, and I wonder how to display the status of a specific management zone ? We set up a token for a specific management zone, but it does not work (rotation of blue leds on top & bottom)
Thanks for your help

**Comment by helmut-s** [comment]
There actually is no code change on the UFO side needed to support this kind of tokens. Does it work with any other token?
For debugging purposes you could also plug it into a computer usb port and open the serial device that appears in a serial monitor (e.g putty or termite)

**Comment by qdengr10** [comment]
Thanks for replying @helmut-s 
Access Token works, no probem.
I found the real problem :  we can not use the endpoint /problem/status on the API V1 (the one used in the code of UFO)) with Personal Access Token because it needs "DataExport" scope and it's not in the list of supported scopes.

It would be nice to add DataExport in supported scopes by PAT ; it will solve this issue and add features! :)
I can open an issue on github if needed

**Comment by helmut-s** [comment]
Thanks for the analysis. I can confirm that the UFO needs to switch to the V2 API in order to make use of the PATs. Unfortunately I cannot confirm a timeline when this will be available as the esp SDK progressed a lot in the meantime and testing all the existing functionality (especially the different Wifi connection methods) will take quite some time...

**Comment by wolfgangB33r** [comment]
@qdengr10 I did look into this request and I am happy to share that I was able to switch the current ufo firmware to use the v2 problem API instead and therefore now allows the use of PAT tokens. Those tokens also respect the users management zones as you already mentioned. 
We did not merge this firmware with the official repository, but I did implemented it in a cloned repository, which you can find here:
https://github.com/wolfgangB33r/ufo-esp32
If you flash the firmware from my repository, the new PAT tokens should work without any change in the UFO functionality.

Best greetings,
Wolfgang

**Comment by qdengr10** [comment]
@wolfgangB33r thank you very much! We will try this & keep you informed ;-)

**Comment by wolfgangB33r** [comment]
@qdengr10 yes pls keep me in the loop if it works for you. On my UFO the PAT token now works as expected with problems v2 read access scope.
